pkg/apis/sedna/v1alpha1/federatedlearningjob_types.go

```diff
@@ -39,25 +39,46 @@ type FederatedLearningJob struct {
 type FLJobSpec struct {
 	AggregationWorker AggregationWorker `json:"aggregationWorker"`
 	TrainingWorkers   []TrainingWorker  `json:"trainingWorkers"`
+	PretrainedModel   PretrainedModel   `json:"pretrainedModel,omitempty"`
+	Transmitter       Transmitter       `json:"transmitter,omitempty"`
+}
+
+// Transmitter describes a transmitter
+type Transmitter struct {
+	S3Transmitter *S3Transmitter    `json:"s3,omitempty"`
+	WSTransmitter map[string]string `json:"ws,omitempty"`
```

Comment:
Define a specified struct for `WSTransmitter`
```suggestion
	ws WSTransmitter  `json:"ws,omitempty"`
```